### DISCUSSION TITLE
disabling extension

### DISCUSSION THREAD

**Discussion by mariannevanharten** [protz/thunderbird-conversations#1632] [Q&A]
Recently I installed this extension.
My expectations of these extension have not been fulfilled, so I disabled and removed the extension.
After restart of the application, Thunderbird is not displaying messages as before. Do I have to change other settings in Thunderbird also?

**Comment by Standard8** [comment]
I suspect what you want is to remove the unified folder view, if so, you want View > Folders > All. This is mentioned in the description of the add-on.

If not, it'd help to have a screenshot of the area that isn't right.

**Reply by mariannevanharten** [reply]
I tried that before, but that didn't help. I don't know why.
Today, according to your reply, I retried it and now it's solved.
Thank you very much!
